cli/src/feature-flags.ts

```diff
@@ -0,0 +1,8 @@
+const flags = (process.env.XATA_CLI_FLAGS || '')
```

Comment:
I believe we should have this in `BaseCommand` and allow them be set from flags in a global way too or env variable (WDYT?)

Reply:
I thought that for things like this we don't want them to expose it with flags, so that's why I didn't put it as ocliff flags. And since it's something that can be isolated and the BaseCommand class is pretty big already, I just put it in a different file.

Reply:
We can revisit it anyway in the future if we add more flags like this.

Reply:
I already had one for workers and I added as a hidden flag in the two commands affected.